frontend/rust-lib/dart-ffi/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 name = "dart_ffi"
 # this value will change depending on the target os
 # default static lib
-crate-type = ["staticlib"]
+crate-type = ["staticlib", "cdylib"]
```

Comment:
If cdylib is not specified it will not build the SDK with the .so extension. Would maybe need to modify the config in Makefile.toml

Reply:
> If cdylib is not specified it will not build the SDK with the .so extension. Would maybe need to modify the config in Makefile.toml

![image](https://user-images.githubusercontent.com/86001920/184756335-6688876c-1fdf-44d4-9656-0527f26b20ba.png)
Check out the frontent/Makefile.toml, the CRATE_TYPE will be modified on a different platforms.